www/polj10/sp1/nastroje/eedice.html

```diff
@@ -0,0 +1,724 @@
+                            style="fill:#fff;" />
+                    </g>
+                </svg>
+            </button>
+            <a href="../index.html" class="logo-link">
+                <img class="logo" src="../img/Ridics_Full_Pack.svg" alt="RIDICS">
+            </a>
+            <div id="navigation">
+                <div class="navigation-ridics-logo">
+                    <img src="../img/Ridics_Full_Pack_Dark.svg" alt="RIDICS logo">
+                </div>
+                <nav id="main-nav" aria-label="Hlavní navigace">
+                    <ul>
+                        <li>
+                            <a href="..//index.html">Úvod</a>
+                        </li>
+                        <li><a href="..//nastroje.html">Nástroje</a></li>
+                        <!-- <li><a href="..//citace.html">Jak citovat</a></li> -->
+                        <li><a href="..//kontakt.html">Kontakt</a></li>
+                    </ul>
+                </nav>
+            </div>
+
+        </header>
+        <main>
+            <div class="poster">
+                <ul class="breadcrumbs">
+                    <li><a href="../nastroje.html">Nástroje</a></li>
+                    <li>&gt; <a href="#" onclick="location.reload();">eEdice (Nápověda)</a></li>
+                </ul>
+                <div class="clear"></div>
+                <h1>eEdice</h1>
+            </div>
+
+            <h2>
+                O nástroji
+            </h2>
+            <p>
+                Nástroj eEdice je software, který slouží k označování relevantních částí textů psaných historickou
+                češtinou a k vytváření sémantické anotace pro potřeby automatického zpracování dalšími nástroji,
+                zejména při převodu do formátu XML podle standardu TEI P5 a do formátu vertikály pro korpusový manažer.
+            </p>
+            <p>
+                Tento nástroj se skládá z doplňku textového editoru Microsoft Office Word 2010 a šablony (eEdice.dotx).
+                Doplněk a šablona jsou distribuovány jako instalační balíček (eEdice.msi). Aktuální verze instalačního
+                balíčku je k dispozici ke stažení na webu. Instalační balíček ke svému spuštění nevyžaduje
+                administrátorská práva, což zjednodušuje distribuci na pracovištích s centralizovanou správou IT.
+            </p>
+            <h2>Instalace</h2>
+            <p>
+                Po spuštění instalačního balíčku se otevře průvodce instalací. Pokud uživatel stáhl instalační balíček
+                z internetu, pak se ještě před instalací pravděpodobně objeví upozornění <i>SmartScreen</i> systému
+                Windows.
+            </p>
+            <figure class="floating-img">
+                <img src="/nastroje/img/smart-screen1.png" alt="Obrázek č.1 - Upozornění filtru SmartScreen">
```

Comment:
relativni cesta, kvuli tomu hle zadny obrazek na teto strance nevidim

Reply:
Damn you VS Code. Opraveno